src/state.cairo

```diff
@@ -127,7 +117,7 @@ pub struct TxIn {
     /// The reference to the previous output that is being used as an input.
     pub previous_output: OutPoint,
     /// The witness data for transactions.
-    pub witness: Span<ByteArray>,
+    pub witness: @ByteArray,
```

Comment:
Witness is a list of items to be pushed onto stack, they can be of different size so you need delimiters

Reply:
I see. What do you suggest? Keeping `Span<ByteArray>` ?

Reply:
Yes